test/stop-operator.sh

```diff
@@ -45,32 +48,74 @@ function delete_olm_operator() {
 
 function delete_operator() {
   DEPLOY_DIRECTORY="${REPO_DIRECTORY}/deploy/operator"
-  deploy="${DEPLOYMENT_DIRECTORY}/pmem-csi-operator.yaml"
+  deploy="${DEPLOY_DIRECTORY}/pmem-csi-operator.yaml"
+
+  tmpdir=$(mktemp -d)
+  trap "rm -rf $tmpdir" SIGTERM SIGINT EXIT
+
+  cat > ${tmpdir}/pmem-csi-operator.yaml <<EOF
+$(cat "${deploy}")
+EOF
+
+  cat > ${tmpdir}/kustomization.yaml <<EOF
+resources:
+- pmem-csi-operator.yaml
+$($keep_namespace && echo -n \
+'patchesStrategicMerge:
+- patch.yaml'
+)
+EOF
+
+  if $keep_namespace ; then
+  cat > ${tmpdir}/patch.yaml <<EOF
```

Comment:
nit: indention of "cat" is the same as "if".

Reply:
Fixed.